eth2util/deposit/deposit.go

```diff
@@ -186,3 +191,40 @@ type depositDataJSON struct {
 	NetworkName           string `json:"network_name"`
 	DepositCliVersion     string `json:"deposit_cli_version"`
 }
+
+// VerifyDepositAmounts verifies various conditions about partial deposits rules.
+func VerifyDepositAmounts(amounts []eth2p0.Gwei) error {
+	if len(amounts) == 0 {
+		return nil
```

Comment:
if no amount is specified are we going with a single deposit of 32 ETH? If yes can we add a comment here.

Reply:
Yes, this is planned for the next iterations in the following PRs.